### ISSUE TITLE
Per Gamemode colors

### ISSUE THREAD

**Issue by eckshton** [absolllute/Mega-Hack-Pro-Future#725]
I love my icons with all white glow but it makes my wave look kinda dumb (cuz white is also my primary)

it'd be sick if i could set just my wave glow or wave colors to be different

**Comment by absoIute** [comment]
Considering a Garage overhaul for v7.2 with this

**Comment by eckshton** [comment]
yeah, that's a good idea. would be sick to have rgb wheels in the garage instead of editing the values (although that should still be a thing). i currently do my colors in a separate program then put them in as values

**Comment by Brittank88** [comment]
> yeah, that's a good idea. would be sick to have rgb wheels in the garage instead of editing the values (although that should still be a thing). i currently do my colors in a separate program then put them in as values

A custom colour square that opens a colour picker akin to what is present in BetterEdit would be amazing, tbh.

Could also have another colour square that is RGB and opens up an in-game RGB configuration menu.

**Comment by absoIute** [comment]
Current plan is
- all existing icons + solid colours
- custom colours
- existing icon effects
- optional separate icons & colours for gamemodes/p1/p2
- access in other menus

It will also supersede the Icon Effects tab so will probably just remove that in the same update
